src/Components/Dashboard.js

```diff
@@ -0,0 +1,5 @@
+const DashboardComponent = () => {
+  return <div>It is located on the dashboard</div>;
+};
+
+export default DashboardComponent;
```

Comment:
lets rename it Dashboard, we already know is a component because is inside the components folder.